test/parallel/test-global-console-exists.js

```diff
@@ -1,39 +1,42 @@
 /* eslint-disable required-modules */
-// ordinarily test files must require('common') but that action causes
-// the global console to be compiled, defeating the purpose of this test
 
 'use strict';
 
-const common = require('../common');
+// Ordinarily test files must require('common') but that action causes
+// the global console to be compiled, defeating the purpose of this test.
+
 const assert = require('assert');
 const EventEmitter = require('events');
 const leakWarning = /EventEmitter memory leak detected\. 2 hello listeners/;
 
-common.hijackStderr(common.mustCall(function(data) {
-  if (process.stderr.writeTimes === 0) {
-    assert.ok(data.match(leakWarning));
-  } else {
-    assert.fail('stderr.write should be called only once');
-  }
-}));
+let writeTimes = 0;
 
-process.on('warning', function(warning) {
+process.on('warning', (warning) => {
```

Comment:
Nit: I would remove the `warning` argument as it is not used. Also there is no guarantee that this listener is called. Should we assert that in a `process.on('exit')` listener?